UNITTESTS/stubs/CellularContext_stub.cpp

```diff
@@ -19,6 +19,15 @@
 
 namespace mbed {
 
+CellularContext::CellularContext() : _next(0), _stack(0), _pdp_type(DEFAULT_PDP_TYPE),
```

Comment:
Should be inited in the same order as declared in header?

Reply:
Yes, and they are what I can tell. Please tell which are not in correct order.